locales/de/LC_MESSAGES/volto.po

```diff
@@ -287,12 +287,12 @@ msgstr "Erweiterung erfolgreich aktualisiert"
 #: components/manage/Blocks/Search/schema
 # defaultMessage: Advanced facet?
 msgid "Advanced facet?"
-msgstr ""
+msgstr "Erweiterte Facette?"
```

Comment:
I suggest to avoid question marks in all options and all languages. 
@stevepiercy what's your opinion?

Reply:
We already have "Multiple choices?" and "Show sorting?" and other cases. So I'd say that out of scope of this PR.